.travis.yml

```
@@ -30,7 +30,7 @@ cache:
 script:
   - jabba use "adopt@~1.8.202-08"
   # need to override as the default is to test
-  - sbt -jvm-opts .jvmopts-travis ";++$TRAVIS_SCALA_VERSION update ;++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues ;++$TRAVIS_SCALA_VERSION test:compile"
+  - sbt -jvm-opts .jvmopts-travis ";++$TRAVIS_SCALA_VERSION update mimaReportBinaryIssues ;++$TRAVIS_SCALA_VERSION test:compile"
```

```suggestion
  - sbt -jvm-opts .jvmopts-travis ";++$TRAVIS_SCALA_VERSION update ;++$TRAVIS_SCALA_VERSION mimaReportBinaryIssues ;++$TRAVIS_SCALA_VERSION test:compile ;++$TRAVIS_SCALA_VERSION validateCompile"
```

Lost an `;++$TRAVIS_SCALA_VERSION` and the `validateCompile`.